packages/ui-extensions-react/src/surfaces/checkout/render.tsx

```diff
@@ -83,6 +83,7 @@ export function reactExtension<Target extends RenderExtensionTarget>(
  */
 export function render<Target extends RenderExtensionTarget>(
   target: Target,
+
```

Comment:
This space is odd. In `checkout-web` there is an [ignore rule](https://github.com/Shopify/checkout-web/blob/9334b922f8047bd49c8f48b3b02267a20722430a/packages/ui-extensions-react/src/surfaces/checkout/render.tsx#L89). How did that get stripped out? 🤔

Reply:
Maybe the way we should solve this is a follow-up to not use the `ApiForRenderExtension` target anyhow, since that was marked as deprecated?